libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/utils/ParsedGradleVersion.kt

```diff
@@ -56,4 +57,12 @@ internal data class ParsedGradleVersion(val major: Int, val minor: Int) : Compar
 
 fun isGradleVersionAtLeast(major: Int, minor: Int) =
     ParsedGradleVersion.parse(GradleVersion.current().version)
-        ?.let { it >= ParsedGradleVersion(major, minor) } ?: false
\ No newline at end of file
+        ?.let { it >= ParsedGradleVersion(major, minor) } ?: false
+
+fun isConfigurationCacheAvailable(gradle: Gradle) =
```

Comment:
:x: you should not need that check at all, see my other comments at usage sites

Reply:
I need an ability to check configuration cache because statistic listeners don't work for it right now and it was moved to another issue.

Reply:
I see, deferring implementation of non essential listeners like statistic collection sounds like a good tradeoff. Thanks for the information.